src/extensions/replay/sessionrecording.ts

```diff
@@ -1185,6 +1186,8 @@ export class SessionRecording {
                     $snapshot_data: snapshotBuffer.data,
                     $session_id: snapshotBuffer.sessionId,
                     $window_id: snapshotBuffer.windowId,
+                    $lib: 'web',
+                    $lib_version: Config.LIB_VERSION,
```

Comment:
I noticed the web uses the `v{version}` pattern and mobile only `{version}, not a problem I guess?

Reply:
ah, yep, it's just a bad mock.. updated to avoid confusion!